src/components/shared/Popover.js

```diff
@@ -133,13 +133,15 @@ class Popover extends Component {
     const arrow = this.getPopoverArrow(orientation, targetMid);
 
     return (
-      <div
-        className={classNames("popover", { up: orientation === "up" })}
-        onMouseLeave={this.onMouseLeave}
-        style={{ top, left }}
-      >
-        {arrow}
-        {this.getChildren()}
+      <div>
```

Comment:
can we try and do this without an extra div?

Reply:
I'll try